src/Core/src/Fonts/FontManager.Android.cs

```diff
@@ -171,7 +172,28 @@ public FontSize GetFontSize(Font font, float defaultFontSize = 0)
 			}
 
 			if (OperatingSystem.IsAndroidVersionAtLeast(28))
+			{
+				var fontWeightMap = new Dictionary<string, FontWeight>
+				{
+					{ "-light", FontWeight.Light },
+					{ "-medium", FontWeight.Medium },
+					{ "-black", FontWeight.Black }
+					// Add more styles as needed
+				};
```

Comment:
Can we make this dictionary `static readonly`? Then we wouldn't create a new one on each call.

String keyed dictionaries should also use `StringComparer.Ordinal` (or `StringComparer.OrdinalIgnoreCase` if casing doesn't matter).

Reply:
Good idea! Thanks!